<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>800</width>
                <height>800</height>
            </rect>
        </property>
        <property name="minimumSize">
            <size>
                <width>800</width>
                <height>800</height>
            </size>
        </property>
        <property name="maximumSize">
            <size>
                <width>800</width>
                <height>800</height>
            </size>
        </property>
        <property name="windowTitle">
            <string>Чаты</string>
        </property>
        <property name="windowIcon">
            <iconset>
                <normaloff>../files/app_images/диалоги.png</normaloff>../files/app_images/диалоги.png
            </iconset>
        </property>
        <property name="whatsThis">
            <string/>
        </property>
        <property name="styleSheet">
            <string notr="true">QMainWindow {
                background-color: rgb(228, 249, 255);
                }
                #pushButton, #pushButton_3, #pushButton_7 {
                background-color: rgb(51, 177, 255);
                border-radius: 20px;
                font: 10pt &quot;Arial&quot;;
                color: rgb(255, 255, 255);
                }
                #pushButton:hover, #pushButton_3:hover, #pushButton_7:hover {
                border-radius: 24px;
                background-color: rgb(52, 202, 232);
                }
                #pushButton:pressed, #pushButton_3:pressed, #pushButton_7:pressed {
                border-radius: 28px;
                background-color: rgb(0, 222, 0);
                }
                QLineEdit {
                border-radius: 10px;
                font: bord, Arial, 15px;
                }
                QLabel {font: bold, Arial, 15px;
                }
                QPushButton:pressed {
                background-color: #42d4da;
                border-style: inset;
                }
                QTextBrowser {
                font: bold, Arial, 12px;
                background-color: #e0f5ff;
                border-color: black;
                border-radius: 10px;}
                QStatusBar {
                font: font: bold, Arial, 8px;
                color: red;}
            </string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <widget class="QListWidget" name="listWidget">
                <property name="geometry">
                    <rect>
                        <x>98</x>
                        <y>49</y>
                        <width>700</width>
                        <height>711</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <family>Arial</family>
                        <weight>75</weight>
                        <bold>true</bold>
                    </font>
                </property>
                <property name="styleSheet">
                    <string notr="true">QPushButton {
                        background-color: rgb(51, 177, 255);
                        border-radius: 17px;
                        font: 10pt &quot;Arial&quot;;
                        color: rgb(255, 255, 255);
                        border: 3px solid #ffffff;
                        }
                        QPushButton:hover {
                        border-radius: 19px;
                        background-color: rgb(52, 202, 232);
                        }
                        QPushButton:pressed {
                        border-radius: 20px;
                        background-color: rgb(0, 222, 0);
                        }
                    </string>
                </property>
            </widget>
            <widget class="QPushButton" name="pushButton">
                <property name="geometry">
                    <rect>
                        <x>3</x>
                        <y>3</y>
                        <width>90</width>
                        <height>90</height>
                    </rect>
                </property>
                <property name="whatsThis">
                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Добавить личное
                        сообщение&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                    </string>
                </property>
                <property name="text">
                    <string/>
                </property>
                <property name="icon">
                    <iconset>
                        <normaloff>../files/app_images/Плюс.png</normaloff>../files/app_images/Плюс.png
                    </iconset>
                </property>
                <property name="iconSize">
                    <size>
                        <width>80</width>
                        <height>80</height>
                    </size>
                </property>
            </widget>
            <widget class="QPushButton" name="pushButton_3">
                <property name="geometry">
                    <rect>
                        <x>3</x>
                        <y>95</y>
                        <width>90</width>
                        <height>90</height>
                    </rect>
                </property>
                <property name="styleSheet">
                    <string notr="true">QPushButton#pushButton_3 {
                        background-image: url('C:\Users\mrmur\Desktop\Андрей\Лого — копия.jpg');
                        }
                    </string>
                </property>
                <property name="text">
                    <string/>
                </property>
                <property name="icon">
                    <iconset>
                        <normaloff>../files/app_images/Икона.png</normaloff>../files/app_images/Икона.png
                    </iconset>
                </property>
                <property name="iconSize">
                    <size>
                        <width>75</width>
                        <height>75</height>
                    </size>
                </property>
            </widget>
            <widget class="QPushButton" name="pushButton_5">
                <property name="geometry">
                    <rect>
                        <x>710</x>
                        <y>3</y>
                        <width>88</width>
                        <height>40</height>
                    </rect>
                </property>
                <property name="styleSheet">
                    <string notr="true">#pushButton_5 {
                        background-color: rgb(51, 177, 255);
                        border-radius: 17px;
                        font: 10pt &quot;Arial&quot;;
                        color: rgb(255, 255, 255);
                        }
                        #pushButton_5:hover {
                        border-radius: 19px;
                        background-color: rgb(52, 202, 232);
                        }
                        #pushButton_5:pressed {
                        border-radius: 20px;
                        background-color: rgb(0, 222, 0);
                        }
                    </string>
                </property>
                <property name="text">
                    <string/>
                </property>
                <property name="icon">
                    <iconset>
                        <normaloff>../files/app_images/Лупа.png</normaloff>../files/app_images/Лупа.png
                    </iconset>
                </property>
                <property name="iconSize">
                    <size>
                        <width>50</width>
                        <height>32</height>
                    </size>
                </property>
            </widget>
            <widget class="QLineEdit" name="lineEdit">
                <property name="geometry">
                    <rect>
                        <x>98</x>
                        <y>3</y>
                        <width>601</width>
                        <height>40</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <family></family>
                        <pointsize>10</pointsize>
                        <weight>50</weight>
                        <italic>false</italic>
                        <bold>false</bold>
                    </font>
                </property>
                <property name="text">
                    <string>Поиск</string>
                </property>
            </widget>
            <widget class="QPushButton" name="pushButton_7">
                <property name="geometry">
                    <rect>
                        <x>3</x>
                        <y>187</y>
                        <width>90</width>
                        <height>90</height>
                    </rect>
                </property>
                <property name="text">
                    <string/>
                </property>
                <property name="icon">
                    <iconset>
                        <normaloff>../files/app_images/Настройки.png</normaloff>../files/app_images/Настройки.png
                    </iconset>
                </property>
                <property name="iconSize">
                    <size>
                        <width>80</width>
                        <height>80</height>
                    </size>
                </property>
            </widget>
        </widget>
        <widget class="QStatusBar" name="statusbar">
            <property name="styleSheet">
                <string notr="true">QStatusBar#statusBar {
                    color: red;
                    font: bold;}
                </string>
            </property>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
